app/components/support_interface/provider_sync_courses_toggle_component.rb

```diff
@@ -0,0 +1,29 @@
+module SupportInterface
+  class ProviderSyncCoursesToggleComponent < ActionView::Component::Base
+    include ViewHelper
+
+    def initialize(provider:)
+      @provider = provider
+    end
+
+    def toggle_path
+      if provider.sync_courses?
+        support_interface_disable_provider_course_syncing_path(provider)
```

Comment:
What would be the effect of disabling sync for a provider? If we don't delete existing courses, previously synced ones will get out of date as some point, which might not be obvious. 

Would it be safer to make enabling courses a one-way operation?

Reply:
Yes I think you are right, it should be one way. Will remove the disable syncing option for now.  

I thought about showing the button if and only if there are no associated courses but then you could get into a race condition with a background worker that just started a sync run.